### PR TITLE
Update logo and update to new brand colors

### PR DESCRIPTION
**Pull request by fernahh**
## Why
:nail_care: 

## Result

> Tip: open in new tab

creditcard js a simple credit card validation library in javascript

@ContaAzul/domphysis

### CODE REVIEW

**Comment by cacontinuos** [comment]
@fernahh, thanks for your PR! By analyzing the blame information on this pull request, we identified @caarlos0 and @matheuspoleza to be potential reviewers

**Comment by YosephCode** [comment]
++

File: css/style.css [hunk 1]
```diff
@@ -77,24 +78,40 @@ code {
...
+  margin: 0 auto;
+}
+
 .ccjs-footer {
   margin-top: 44px;
   padding: 34px;
 
+  border-bottom: 1px solid #dddddd;
+
+  color: #9fa2a7;
```
Comment (thiagogodoy) [inline css/style.css]: Remove essas linhas em branco ;)

**Comment by thiagogodoy** [comment]
++

**Comment by matheuspoleza** [comment]
top +1

**Comment by matheuspoleza** [comment]
@fernahh can you change the README image too?

**Pull request merged**